how2meet/routers/events.py

```diff
@@ -0,0 +1,89 @@
+from fastapi import APIRouter, Depends, HTTPException
+from sqlalchemy.orm import Session
+
+from ..db import crud, models, schemas
+from ..db.database import engine, get_db
+
+router = APIRouter(
+    prefix="/events",
+    tags=["events"],
+    responses={404: {"description": "Not found"}}
+)
+
+@router.post("/", response_model=schemas.Event)
+def create_event(event: schemas.EventCreate,
```

Comment:
The fact that this file contains both the route definitions _and_ the controller functions they run, I don't know if I like the enclosing folder being "routers". The name feels a little too narrow. Because this kind of _is_ our backend here, not just the router.

Reply:
Maybe we just call the enclosing folder `api`?